packages/shared/src/scope.ts

```diff
@@ -0,0 +1,85 @@
+export enum PreservedScope {
+  OpenId = 'openid',
+  OfflineAccess = 'offline_access',
+}
+
+export type UserClaim =
+  | 'name'
+  | 'picture'
+  | 'username'
+  | 'role_names'
+  | 'email'
+  | 'email_verified'
+  | 'phone_number'
+  | 'phone_number_verified'
+  | 'custom_data'
+  | 'identities';
+
+/**
+ * Scopes for ID Token and Userinfo Endpoint.
+ */
+export enum UserScope {
+  /**
+   * Scope for basic user info.
+   *
+   * See {@link idTokenClaims} for mapped claims in ID Token and {@link userinfoClaims} for additional claims in Userinfo Endpoint.
+   */
+  Profile = 'profile',
+  /**
+   * Scope for user email address.
+   *
+   * See {@link idTokenClaims} for mapped claims in ID Token and {@link userinfoClaims} for additional claims in Userinfo Endpoint.
```

Comment:
The similar repeated comments can be moved to the comment block above `export enum UserScope`?

Reply:
it's by design

- when typing enum member it won't show parent comments
- js/tsdoc now don't have a way to reuse comments